lib/cluster_build.ml

```diff
@@ -17,7 +17,7 @@ let tail ?buffer ~job build_job =
     | Error (`Capnp e) -> Lwt.return @@ Fmt.error_msg "%a" Capnp_rpc.Error.pp e
     | Ok ("", _) -> Lwt_result.return ()
     | Ok (data, next) ->
-      Option.iter (fun b -> Buffer.add_string b data) buffer;
+      Stdlib.Option.iter (fun b -> Buffer.add_string b data) buffer;
```

Comment:
I wonder if it might not be a better idea to disable implicit transitive dependencies in our build configuration: https://dune.readthedocs.io/en/stable/reference/dune-project/implicit_transitive_deps.html

I'm not crazy about the fact that adding a dependency for our CLI tool pollutes the global module name space for the entire project XD 

WDYT?

Reply:
Yes, your suggestion might be a better way to go about this. But, I ended up completely removing the commit. It is an ancient commit that was leftover from my experiments when I was trying to use the extracted code as a library, instead of a CLI tool.